_posts/2023-05-01-customize-your-openmage.md

```diff
@@ -81,6 +81,22 @@ Note, you can also have the patch downloaded at runtime by defining it as a url
 
 If it doesn't apply cleanly, apply the patch using `git apply` and fix the conflicts and then output the changes as a local patch file using one of the methods below.
 
+### symplify/vendor-patches
+
+If you are using OpenMage as a project dependency, you can use `symplify/vendor-patches` to generate a patch file easily. This tool automatically
+updates your composer.json for you. Note, in the case of modifying OpenMage core code installed as a project dependency you will need to modify the
+files in the "vendor" directory rather than the `magento-root-dir` directory (e.g. "htdocs").
+
+Here is an example generating patches for `app/Mage.php`:
+
+```sh
+composer require symplify/vendor-patches --dev
```

Comment:
Line can be removed. Was added as dev-req with zf1f.

Reply:
Hmm, it was necessary when I tried it.. I think to use it in a project which has openmage as a dependency I think it is still required because dev dependencies of packages are not installed by default. The guide is written for users which would be using openmage as a dependency, not the openmage devs.